src/jsBundle.js

```diff
@@ -2,53 +2,62 @@
 
 const polyfill = require("array.prototype.flatmap");
 polyfill.shim();
-const execa = require("execa");
 const fs = require("fs").promises;
-const path = require("path");
 const { createEntryFile } = require("./create-entry-file-js");
 const { parseModulesParameter } = require("./parse-modules-parameter");
-const { Entrypoint } = require("./modules/Entrypoint");
-const { GET } = require("./modules/SolveType");
-const { SystemCache } = require("./modules/SystemCache");
 const util = require("util");
 const rimraf = require("rimraf");
 const rmrf = util.promisify(rimraf);
+const createPackageJsonFile = require("./modules/create-package-json-file");
+const installDependencies = require("./modules/install-dependencies");
+const createJavaScriptBundle = require("./modules/create-javascript-bundle");
+const { SolveFailure } = require("./modules/SolveFailure");
+const { UserException, FormatException } = require("./modules/HOME");
 
-console.clear();
 const jsBundle = async (querystring = {}) => {
   await fs.mkdir("/tmp/bundle/", { recursive: true });
   const bundleLocation = await fs.mkdtemp("/tmp/bundle/");
 
   try {
-    await rmrf(`/Users/jake.champion/.jake-cache`);
```

Comment:
oh no, but that's where i was keeping my local jakes

Reply:
😂 Sorry Chee